src/ontology/mondo-edit.obo

```diff
@@ -570873,7 +570873,7 @@ synonym: "coat color albinism, oculocutaneous type IV, Japanese medaka" RELATED
 xref: OMIA:001821-8090 {source="MONDO:equivalentTo"}
 is_a: MONDO:1011321 {source="OMIA:001821-8090", source="https://orcid.org/0000-0002-5002-8648"} ! hereditary disease, non-human animal
 is_a: MONDO:1012579 {source="OMIA:001821-8090", source="https://orcid.org/0000-0002-5002-8648"} ! oculocutaneous albinism, non-human animal
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/ncbigene/100049347 {source="OMIA:001821-8090", source="https://orcid.org/0000-0002-5002-8648"}
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/ncbigene/100049347 {source="OMIA:001821-8090", source="https://orcid.org/0000-0002-5002-8648"} ! slc45a2
```

Comment:
Confirmed that this ncbigene UID does have a lowercase label: https://www.ncbi.nlm.nih.gov/gene/?term=100049347